src/NewTools-Playground-Tests/StPlaygroundBindingsPresenterTest.class.st

```diff
@@ -0,0 +1,154 @@
+Class {
+	#name : 'StPlaygroundBindingsPresenterTest',
+	#superclass : 'StPlaygroundBaseTest',
+	#instVars : [
+		'owner'
+	],
+	#category : 'NewTools-Playground-Tests',
+	#package : 'NewTools-Playground-Tests'
+}
+
+{ #category : 'accessing' }
+StPlaygroundBindingsPresenterTest >> classToTest [
+
+	^ StPlaygroundBindingsPresenter.
```

Comment:
No need to add points at the end of a method. Don't hesitate to do CMD/Ctrl + Shift + f on your methods to format them.

```suggestion
	^ StPlaygroundBindingsPresenter
```